lib/automatic/api_client.rb

```diff
@@ -0,0 +1,95 @@
+require 'automatic/api_client/trips'
+require 'automatic/api_client/vehicles'
+
+module Automatic
+  class ApiClient
```

Comment:
yeah since these same methods are class-level in one and instance in another...does creating a module which is extended in `Automatic::Client` and included in `Automatic::ApiClient` make sense?